pkgs/top-level/all-packages.nix

```
@@ -5553,6 +5553,10 @@ with pkgs;
 
   klog = qt5.callPackage ../applications/radio/klog { };
 
+  knxd = callPackage ../servers/knxd {
+    fmt = fmt_9;
+  };
```

```suggestion
  knxd = callPackage ../servers/knxd { };
```

this usually breaks cross